llvm/lib/Target/AArch64/AsmParser/AArch64AsmParser.cpp

```diff
@@ -3896,6 +3954,7 @@ static const struct Extension {
     {"f16mm", {AArch64::FeatureF16MM}},
     {"f16f32dot", {AArch64::FeatureF16F32DOT}},
     {"f16f32mm", {AArch64::FeatureF16F32MM}},
+    {"poe2", {AArch64::FeatureS1POE2}},
```

Comment:
Should these:
{"tea", {AArch64::FeatureTEV}},
{"btie", {AArch64::FeatureBTIE}},
be added too?